File: example/lib/models/AppInfo.dart
import 'package:json_annotation/json_annotation.dart';

part 'AppInfo.g.dart';

@JsonSerializable()
class AppInfo {
    AppInfo();

    bool isSystem;
    String name;
    String packageName;
    String packagePath;
    bool isRunning;
    num versionCode;
    String versionName;
    
    factory AppInfo.fromJson(Map<String,dynamic> json) => _$AppInfoFromJson(json);
    Map<String, dynamic> toJson() => _$AppInfoToJson(this);
}

File: android/src/main/kotlin/com/sctuopuyi/packageinfos/bean/AppInfo.kt
package com.sctuopuyi.packageinfos.bean

import android.graphics.drawable.Drawable
import com.google.gson.annotations.SerializedName

data class AppInfo(
        @SerializedName("packageName") val packageName: String,
        @SerializedName("name") val name: String,
        @SerializedName("packagePath") val packagePath: String,
        @SerializedName("versionName") val versionName: String,
        @SerializedName("versionCode") val versionCode: Int,
        @SerializedName("isSystem") val isSystem: Boolean,
        @SerializedName("isRunning") val isRunning: Boolean
)